activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -409,14 +409,8 @@ def dump_filename(namespace, format = ActiveRecord::Base.schema_format)
         ENV["SCHEMA"] || File.join(ActiveRecord::Tasks::DatabaseTasks.db_dir, filename)
       end
 
-      def cache_dump_filename(namespace)
-        filename = if namespace == "primary"
-          "schema_cache.yml"
-        else
-          "#{namespace}_schema_cache.yml"
-        end
-
-        ENV["SCHEMA_CACHE"] || File.join(ActiveRecord::Tasks::DatabaseTasks.db_dir, filename)
+      def cache_dump_filename(namespace, schema_cache_path = nil)
```

Comment:
What do you think of renaming `namespace` in here to `spec_name` for consistency with other places we're using this value?

Reply:
Yeah, good call.

Reply:
Hm. In the context of the filename, I'm not actually sure I want to call it `spec_name`. The `spec_name` is specific to the database config, and here we don't see that it came from the db config, so `spec_name` would be more confusing unless you happen to know off the top of your head where this is coming from.